specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1796,25 +1798,67 @@
       "type": "object",
       "required": [
         "relationType",
-        "bidirectional",
-        "source",
-        "target"
+        "entities"
       ],
       "properties": {
         "relationType": {
           "description": "Type of relation. Examples include: `DosageOfMedication` or 'FrequencyOfMedication', etc.",
           "type": "string"
         },
-        "bidirectional": {
-          "description": "If true the relation between the entities is bidirectional, otherwise directionality is source to target.",
-          "type": "boolean"
-        },
-        "source": {
-          "description": "Reference link to the source entity.",
+        "entities":{
```

Comment:
We should add a `description` field for the `HealthcareRelation` type. Explaining that every relation is an entity graph of a certain `entityType`, where all entities are connected and have specific roles within the relation context.